### PR TITLE
Feature/new tango ros release

### PR DESCRIPTION
**Pull request by PerrineAguiar**
This PR adds artifacts from the new release of TangoRosStreamer (v1.3.0).

### CODE REVIEW

**Comment by PerrineAguiar** [comment]
@jubeira ptal

File: eu/intermodalics/tango_nodelet_manager/maven-metadata.xml [hunk 1]
```diff
@@ -3,12 +3,10 @@
   <groupId>eu.intermodalics</groupId>
   <artifactId>tango_nodelet_manager</artifactId>
   <versioning>
-    <release>1.2.0</release>
+    <release>1.3.0</release>
     <versions>
-      <version>1.1.0</version>
-      <version>1.1.1</version>
-      <version>1.2.0</version>
```
Comment (jubeira) [inline eu/intermodalics/tango_nodelet_manager/maven-metadata.xml]: This one looks a bit weird to me. This file generally keeps the old versions as well, and writes the latest one inside the `release` tag. I don't remember exactly how to do that, but I think that's the way it should be (at least it was done like that until now).

File: eu/intermodalics/tango_ros_common/maven-metadata.xml [hunk 2]
```diff
@@ -3,12 +3,10 @@
   <groupId>eu.intermodalics</groupId>
   <artifactId>tango_ros_common</artifactId>
   <versioning>
-    <release>1.2.0</release>
+    <release>1.3.0</release>
     <versions>
-      <version>1.1.0</version>
-      <version>1.1.1</version>
-      <version>1.2.0</version>
```
Comment (jubeira) [inline eu/intermodalics/tango_ros_common/maven-metadata.xml]: same as above

**Comment by PerrineAguiar** [comment]
@jubeira Thanks for your review. I addressed your comments.

**Comment by PerrineAguiar** [comment]
@jubeira Ok, no problem.

**Comment by adamantivm** [comment]
I fear @PerrineAguiar may come to nag me in person

**Comment by PerrineAguiar** [comment]
@adamantivm Haha, I did not think about this ^^
Thanks a lot!

**Pull request merged**